core-build/gulp-core-build-typescript/src/defaultTslint.json

```diff
@@ -13,9 +13,25 @@
     "member-access": true,
     "member-ordering": [
       true,
-      "public-before-private",
-      "static-before-instance",
-      "variables-before-functions"
+      {
+        "order": [
+          "public-static-field",
+          "protected-static-field",
+          "private-static-field",
+          "public-instance-field",
+          "protected-instance-field",
+          "private-instance-field",
+          "public-static-method",
+          "protected-static-method",
+          "private-static-method",
+          "public-constructor",
+          "public-instance-method",
+          "protected-constructor",
+          "protected-instance-method",
+          "private-constructor",
+          "private-instance-method"
```

Comment:
It's as similar to our existing order as possible.

---
In reply to: [129392775](https://github.com/Microsoft/web-build-tools/pull/276#discussion_r129392775) [](ancestors = 129392775)